nixos/modules/services/networking/spacecookie.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.spacecookie;
+  defaultUser = "spacecookie";
+  configFile = pkgs.writeText "spacecookie.json" (lib.generators.toJSON {} {
+    hostname = cfg.hostname;
+    port = cfg.port;
+    user = cfg.user;
+    root = cfg.root;
```

Comment:
We run as a restricted user via systemd already anyways, so the code dropping privileges isn't really needed.

Can we not have it in the configuration file?

Reply:
Unfortunately, the config file wouldn't parse that way. It is kinda silly though, that this does not work currently, so I'll make a new release where this is optional. We currently have to wait on the new `hackage-packages.nix` anyways...

Reply:
So we can set some bogus values here, until there's the new release of `spacecookie` in nixpkgs?

Reply:
Yeah, that'd be possible.

Reply:
Sadly not, due to a bug it doesn't work as intended currently and would always call `setuid` :/